components/console/locales/extracted/en.json

```diff
@@ -121,6 +144,8 @@
   "dialog.about.midaz.license": "License",
   "dialog.about.midaz.terms": "Terms of Use",
   "dialog.about.midaz.version": "Version {version}",
+  "dialog.delete.confirmLabel": "Yes, delete it",
```

Comment:
This key looks to generic, can you change it to "applications.dialog.delete..confirmLabel"?
Do the same to the others